tests/correct.py

```diff
@@ -60,5 +60,9 @@ def modify_list(num_list):
     assert get_dictionary_value({'a': 1, 'b': 2, 'c': 3}, 'd') == 'The available options are - a, b, c'
 
     assert modify_list([1, 2, 3, 4, 5]) == [1, 3, 5, 7, 9]
+    assert modify_list([1]) == [1]
     assert modify_list(['hello', 10]) == 'List contains type that is not an integer at position 0'
     assert modify_list([]) == -1
+    assert modify_list([0.1, 'Kelly', 'h']) == 'List contains type that is not an integer at position 0'
```

Comment:
We need to have these assertions as part of the test because in the test we check the student's solution against these cases.

Reply:
Oh right, sorry let me fix that